src/firmware/app/control/physbot.c

```diff
@@ -16,8 +16,8 @@ PhysBot app_physbot_create(const FirmwareRobot_t *robot, float *destination,
         .disp = dot2D(major_vec, dr), .vel = dot2D(major_vec, v), .accel = 0, .time = 0};
     const Component minor = {
         .disp = dot2D(minor_vec, dr), .vel = dot2D(minor_vec, v), .accel = 0, .time = 0};
-    const Component rot = {.disp = min_angle_delta(
-                               app_firmware_robot_getOrientation(robot), destination[2])};
+    const Component rot = {.disp =
+                               destination[2] - app_firmware_robot_getOrientation(robot)};
```

Comment:
This change seems quite confusing to me. Why don't we want the minimum displacement (to avoid the robot rotating the wrong way round)?

Reply:
I'm guessing it has something to do with assumptions we're making about the output of the trajectory planner, but I'd really strongly recommend against relying on assumptions between these totally independent pieces of code.

Reply:
this is so that the robot will spin, i.e. if destination is 510deg and current orientation is 180deg then it will spin counterclockwise, rather than clockwise to 510-360=150deg

Reply:
Hmm, but aren't the increments coming out of the trajectory planner really small? And won't this break some cases where we might want it to spin clockwise?

Reply:
essentially I'm trying to shift the responsibility of determining which way to spin to the user of this function, so if `destination[2] - app_firmware_robot_getOrientation(robot)` is negative, then it will spin clockwise.